chromadb/quota/__init__.py

```diff
@@ -11,7 +13,14 @@ def __init__(self, system: System) -> None:
         super().__init__(system)
 
     @abstractmethod
-    def enforce(self) -> None:
+    def set_context(self, context: Dict[str, Any]) -> None:
+        """
+        Sets the context for a given request.
+        """
+        pass
+
+    @abstractmethod
+    def enforce(self, action: str, tenant: str, **kwargs: Dict[str, Any]) -> None:
```

Comment:
Technically the explicitly named `tenant` parameter isn't necessary because it will always be included within the function arguments we send in `kwargs`. But I thought for clarity and understandability it made sense to explicitly name it.